modules/dedicated-configuring-your-application-routes.adoc

```diff
@@ -5,7 +5,7 @@
 [id="dedicated-configuring-your-application-routes_{context}"]
 = Configuring your application routes
 
-When your cluster is provisioned, an AWS elastic load balancer (ELB) is created
+When your cluster is provisioned, an AWS elastic load balancing (ELB) load balancer is created
```

Comment:
@dfitzmau AWS does not use the term "AWS elastic load balancing". The full service name is always "Elastic Load Balancing" by itself, without "AWS". The name should be capitalized also. 

Suggested revision: "When your cluster is provisioned, an Elastic Load Balancing  (ELB) load balancer is created.